### ISSUE TITLE
[Feature] Skill Improvements

### ISSUE THREAD

**Issue by PendragonII** [TeamLapen/Vampirism#475]
## What would you like to have changed or added to Vampirism
Options to devote skills points onto improving unlocked skills instead of unlocking a new skill:
When players level up and unlock a new skill point they can either unlock a new skill or spend the new skill point onto an already unlocked skill improving it.

Example: A level 5 Vampire Hunter may want to spend their skill point on the Disguise Skill again rather than unlock another skill bringing the Disguise Skill to Tier II allowing the Hunter to place blocks while in Disguise.

This could add more levels, perhaps.
## Does this make the mod experience more balanced?
As writting in Issue #136 , the Hunter Disguise Skill doesn't allow Hunters to place blocks because it will be unbalanced. To balance this example - Vampires with the Bat Skill Tier II may have more health while being in bat mode.

TL;DR: This can be Balanced and Unbalanced.

**Comment by PendragonII** [comment]
This issue is a derivative of #412

**Comment by PendragonII** [comment]
I would personally suggest for higher levels (16+) something different from Barons. Perhaps:
Achievements such as "Conquistador - Capture 3 Villages" // "Ending of the Oligarchy - Slay 3 Level 6 Barons".
Or repeating level up requirements for level 15 to progress levels (if a Hunter/Vampire is level 18, they must meet requirements for level 16 to progress to level 19, a repeating requirement)
These are suggestions for more levels for more progression.

**Comment by Cheaterpaul** [comment]
Idea:
you unlock the skills normally, but can develop them further if you get achievements like you suggest.

**Comment by maxanier** [comment]
This is now implemented in an alternative way based on the accessories which offer skill up/sidegrades.
The "achievements" are more are less implemented in form of the faction representative tasks.
Allowing higher levels causes issues with balancing, I think. We can't just buff the player stats endlessly
